docs/red_eyes_mainnet_tutorial.rst

```diff
@@ -0,0 +1,142 @@
+
+.. _join-token-network:
+
+Joining a token network
+==============================
+The first thing we need to do is to join a token network. In this case we want to join the Raiden token (`RDN <https://etherscan.io/token/0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6>`_) network.
+
+*Note*: 1 RDN == 2**18 Rai. For the sake of readability and simplicity all token values in this tutorial are denominated in Rai and not RDN.
+
+In order to do so, we need the address of the token and the initial amount of tokens that we want to join the network with::
+
+    PUT /api/v2/connections/0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6
+
+TODO: check correctness of API
+With the payload::
+
+    {
+        "funds": 20000
+    }
+
+This automatically connects our node to 5(TODO) other nodes in the network with 20000 / 5 RDN tokens in each channel.
+
+We're now ready to start sending RDN tokens using the Raiden Network.
+
+In case we know of a specific node in the network that we will do frequent transfers with, we can open a channel directly yo this node by doing the following::
+
+    PUT /api/v2/channels
+
+With the payload::
+
+    {
+        "partner_address": "0x61c808d82a3ac53231750dadc13c777b59310bd9",
+        "token_address": "0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6",
+        "balance": 2000,
+        "settle_timeout": 600
+    }
+TODO: Update to have correct values
+
+At this point the specific value of the ``balance`` field isn't too important, since it's always possible to :ref:`deposit more tokens <topping-up-a-channel>` to a channel if need be.
+
+Successfully opening a channel returns the following information::
+
+    {
+        "channel_identifier": "0x2a65aca4d5fc5b5c859090a6c34d164135398226",
+        "partner_address": "0x61c808d82a3ac53231750dadc13c777b59310bd9",
+        "balance": 2000,
+        "state": "opened",
+        "settle_timeout": 60
+    }
+
+TODO: check correctness
+
+.. _doing-payments:
+
+Payments
+========
+Now that we have joined a token network, we can start making payments. For this, we need the address of the RDN token and the address of the recipient of the payment::
+
+    POST /api/v2/transfers/0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6/0x61c808d82a3ac53231750dadc13c777b59310bd9
+
+In the payload we specify the amount of RDN tokens of the payment::
+
+    {
+        "amount": 42
+    }
+TODO: Check for correctness. What denomination are we using for amount?
+
+In this specific case the payment goes directly to one of our channel partners, since we have an open channel with ``0x61c808d82a3ac53231750dadc13c777b59310bd9``. If we specify an address that we do not have a direct channel with, the Raiden Network finds a path to the target address and use mediated transfers to make a payment from our address to the target address.
+
+It's as simple as that to do payments using the Raiden Network.
```

Comment:
Here it may be hard to see at first read how simple it really was, as the explanations of the steps so far were really detailed. Maybe we can sum up here what happened so far: "So we have done the first payment after just two API calls - one to join a token network and one to transfer. The third call we did to open the direct channel was optional."